package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

You should maybe update your local master branch :slightly_smiling_face: 
```suggestion
  "version": "2.3.2",
```

---

And generate again the package-lock.json

---

Never mind we choose what fix we want to release first

---

dang

---

@alexandrebouthinon > err... are you sure?
https://github.com/kuzzleio/kuzzle/blob/master/package.json

---

Talking about this on Slack https://kaliop.slack.com/archives/G4LN18NLQ/p1593763809492900?thread_ts=1593762619.490900&cid=G4LN18NLQ